test/retrieveFile.m

```diff
@@ -0,0 +1,23 @@
+function retrieveHTMLFile(filepath, filename)
```

Comment:
Please include a header describing what the function does and its expected inputs.